docs/docs/en/user-guide/adapters/fastapi/index.md

```diff
@@ -158,9 +159,9 @@ The following section presents high-level architecture diagrams for the two avai
 
         Before we examine the code:
 
-        - The below example uses a simple HTML document with JavaScript, all in a single string and served directly from the FastAgency FastAPI app for simplicity.
-        - This approach is not suitable for production but ideal for demonstrating core concepts.
-        - In a real-world scenario, you'd use a separate frontend built with a framework like React or Vue.js.
+        - The below example uses a **simple HTML with JavaScript**, all in a single string and served directly from the FastAgency FastAPI app for **simplicity**.
+        - This approach is **not suitable for production** but ideal for demonstrating core concepts.
+        - In a real-world scenario, you'd use a separate frontend built with a framework like **React or Vue.js**.
```

Comment:
you can also say that most other languages and frameworks could be using this, like go, java,..